sdk/tests/fixtures/mlflow.py

```diff
@@ -8,7 +8,7 @@ def mock_mlflow(responses, experiment_name, experiment_id, run_id, artifact_uri)
     responses.add(
         method="GET",
         url=f"/api/2.0/mlflow/experiments/get-by-name?experiment_name={quote_plus(experiment_name)}",
-        body=json.dumps({"experiment": {"id": experiment_id, "name": experiment_name}}),
+        body=json.dumps({"experiment": {"id": experiment_id, "name": experiment_name, "lifecycle_stage": "active"}}),
```

Comment:
Added this additional field to the mocked response because subsequent versions of `mlflow` (`>= 1.22.0`) are now expecting an additional field: https://github.com/mlflow/mlflow/commit/0f9d1e04975c69313ad57e4d7d428ab7befad69e#diff-b33420360cc29a5224c8d3080bbfcaa0ac1c8891416137e13be044fcdfdd26f9R123

See issue 5012 in https://github.com/mlflow/mlflow/releases/tag/v1.22.0 for more details.